recipe/meta.yaml

```diff
@@ -36,6 +36,7 @@ test:
     - test -e $PREFIX/lib/libblitz.dylib   # [osx]
     - if not exist %PREFIX%\Library\bin\blitz.dll exit 1  # [win]
     - if not exist %PREFIX%\Library\lib\blitz.lib exit 1  # [win]
+    - if not exist %PREFIX%\Library\lib\pkgconfig\blitz.pc exit 1  # [win]
```

Comment:
You better test for a couple of header files too

Reply:
There is little or no chance these may be missing, however, you are right, it never hurts to check.